content/guards.md

```diff
@@ -18,13 +18,15 @@ As mentioned, **authorization** is a great use case for Guards because specific
 @@filename(auth.guard)
 import { Injectable, CanActivate, ExecutionContext } from '@nestjs/common';
 import { Observable } from 'rxjs';
+import type { Request } from 'express';
+import { validateRequest } from './my-custom-authorization'
```

Comment:
This is quite confusing

Reply:
Could you clarify a bit what is confusing? The reasoning behind the import is that it is missing in the example, but it is used to define the authorization for the assigned endpoints. (Line 30)